.travis.yml

```diff
@@ -25,5 +27,6 @@ jobs:
       script: make test BUILDTAGS=""
 
     - stage: validate
-      script: |
+    
+script: |
```

Comment:
Was this intentional? If I see correctly, this moves the actual script out of the validate step?

Reply:
sorry ,it was my mistake , i have correct this move and ran the job, without matrix poweron host is not running for stages build,Validate,Tint. only Test stage is included  power on arch as per below run. 

refer https://travis-ci.com/github/asellappen/selinux for details.

This was the reason i used matrix to run package for both cpu arch.

please check and pass your inputs.